bodhi-server/bodhi-server.spec

```diff
@@ -83,6 +128,20 @@ install -pm0644 docs/_build/*.1 %{buildroot}%{_mandir}/man1/
 %{python3_sitelib}/bodhi_server-%{pypi_version}-py%{python3_version}.egg-info
 %{_mandir}/man1/bodhi-*.1*
 %{_mandir}/man1/initialize_bodhi_db.1*
+# These excluded files are in the bodhi-consumers package so don't include them here.
```

Comment:
You probably mean `bodhi-composer` in the comment here, no?

Reply:
fixed! thanks!